client/src/components/LeftSidebar/LeftSidebar.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Nav } from "react-bootstrap";
+export default function LeftSidebar(props) {
+  return (
+  
+      <Nav className="col-md-12 d-none d-md-block bg-light sidebar"
+        activeKey="/home"
+        onSelect={selectedKey => alert(`selected ${ selectedKey } `)}
```

Comment:
Wondering why you are alerting the selected key? This component should take a callback in its props (for example `onUpdate`) and call that function when the user selects an item.